src/pages/transcripts.js

```diff
@@ -8,7 +8,7 @@ export default function Transcripts({ data }) {
     .filter((node) => node.childMarkdownRemark.html.trim().length)
     .map((node) => ({
       title: node.childMarkdownRemark.frontmatter.title,
-      path: `/${node.relativeDirectory}/${node.name}`,
+      path: `${node.name}`,
```

Comment:
One suggestion!
```suggestion
      path: node.name,
```

Reply:
Ofcourse! I'll change this tonight (in ~10 hours)

Reply:
You can press "Commit suggestion" to commit automatically